packages/scripts/src/application.ts

```diff
@@ -268,6 +269,28 @@ export class Application {
             });
         });
 
+        if (mode === 'development') {
+            app.get('/render-graph', (req, res) => {
+                const featureName = req.query['feature-name'] as string;
+
+                const visitedFeatures = {} as { [propName: string]: number };
+
+                const links = buildFeatureLinks(features.get(featureName)!.exportedFeature, visitedFeatures, 0);
+
+                const graph = {
+                    nodes: Object.keys(visitedFeatures)
+                        .map((name) => ({ name, id: name, group: visitedFeatures[name] }))
+                        .concat({
+                            name: features.get(featureName)!.exportedFeature.id,
```

Comment:
you already retrieved `features.get(featureName)` earlier. put it in a const and reuse